rd_ui/app/scripts/directives/plotly.js

```diff
@@ -121,6 +121,8 @@
   angular.module('plotly', [])
     .constant('ColorPalette', ColorPalette)
     .directive('plotlyChart', function () {
+      var baseHeight = 250;
```

Comment:
Hello Arik,
Kumar has gone on vacation and I'm going to pick up where he left off.  Which alternative approach would you prefer we take?  Delegating to Plotly sounds more elegant.  What do you think?